index.js

```diff
@@ -188,5 +188,12 @@ const saveEditChanges = (event) => {
 
 };
 
+const valueCheck = (num1, num2) => {
+  if (num1 > num2) {
+    return "Num1 is Bigger then Num2";
+  }
+  return "Num1 is Smaller then Num2";
+}
```

Comment:
**Suggestion:** The function `valueCheck` is not exported. If you want to use it in other modules, consider exporting it. [maintainability]
```suggestion
export const valueCheck = (num1, num2) => {
  if (num1 > num2) {
    return "Num1 is Bigger then Num2";
  }
  return "Num1 is Smaller then Num2";
}
```